docs/setting-up/client/mongodb.md

```diff
@@ -33,140 +64,213 @@ db.getSiblingDB("admin").createRole({
+
+2. Create or add this to the `operationProfiling` section. ([Read more][MONGODB_CONFIG_OP_PROF].)
+
+    ```yml
+    operationProfiling:
+      mode: all
+      slowOpThresholdMs: 200
+      rateLimit: 100
+    ```
+
+    > This is a [YAML](http://yaml.org/spec/) file. Indentation is important.
+
+3. Restart the `mongod` service. (Example for `systemd`.)
+
+    ```sh
+    systemctl restart mongod
+    ```
+
+### Set profiling on the command Line
 
 ```sh
 mongod --dbpath=DATABASEDIR --profile 2 --slowms 200 --rateLimit 100
 ```
 
-Note that you need to specify a path to an existing directory that stores
-database files with the `--dpbath`. When the `--profile` option is set to
-2, `mongod` collects the profiling data for all operations. To decrease the
-load, you may consider setting this option to 1 so that the profiling data
-are only collected for slow operations.
+- `--dbpath`: The path to database files (usually `/var/lib/mongo`).
+- `--profile`: The MongoDB profiling level. A value of `2` tells the server to collect profiling data for *all* operations. To lower the load on the server, use a value of `1` to only record slow operations.
+- `--slowms`: An operation is classified as *slow* if it runs for longer than this number of milliseconds.
+- `--rateLimit`: (Only available with Percona Server for MongoDB.) The sample rate of profiled queries. A value of `100` means sample every 100th fast query. ([Read more][PSMDB_RATELIMIT].)
 
-The `--slowms` option sets the minimum time for a slow operation. In the
-given example, any operation which takes longer than 200 milliseconds is a
-slow operation.
+    > Smaller values improve accuracy but can adversly affect the performance of your server.
 
-The `--rateLimit` option, which is available if you use PSMDB instead
-of MongoDB, refers to the number of queries that the MongoDB profiler
-collects. The lower the rate limit, the less impact on the performance.
-However, the accuracy of the collected information decreases as well.
+### Set profiling in a `mongo` session
 
-### Enabling Profiling in the Configuration File
+In a `mongo` session:
 
-If you run `mongod` as a service, you need to use the configuration file
-which by default is `/etc/mongod.conf`.
+```json
+use admin
+db.setProfilingLevel(2)
+```
 
-In this file, you need to locate the `operationProfiling:` section and add the
-following settings:
+> If you have already [added a service](#add-service), you should remove it and re-add it after changing the profiling level.
 
-```
-operationProfiling:
-   slowOpThresholdMs: 200
-   mode: slowOp
-```
+## Add service
 
-These settings affect `mongod` in the same way as the command line options. Note that the configuration file is in the [YAML](http://yaml.org/spec/) format. In this format the indentation of your lines is important as it defines levels of nesting.
+When you have configured your database server, you can add a MongoDB service with the user interface or on the command line.
```

Comment:
Is one better than the other?  I have been of the mindset (maybe wrong) that CLI gives more control to the user (and also spreads the exporter load out across all clients vs concentrating on PMM server).

Reply:
Good question but no idea how it is seen by users. In previous page refreshes I began the idea of separating CLI/UI steps to cater to all kinds of people. It makes it messy but at least it is complete and readers can see how the CLI commands and options relate to their UI equivalents.